tools/link.py

```diff
@@ -393,10 +393,7 @@ def check_human_readable_list(items):
       passes += ['--pass-arg=asyncify-onlylist@%s' % ','.join(settings.ASYNCIFY_ONLY)]
 
   if settings.MEMORY64 == 2:
-    passes += ['--memory64-lowering']
-
-  if settings.MEMORY64:
-    passes += ['--table64-lowering']
+    passes += ['--memory64-lowering', '--table64-lowering']
```

Comment:
should --table64-lowering go away? I guess we can just roll the passes together into memory64-lowering now?

Reply:
Yes, eventually, but I guess that can/should be a followup.